test/images/mount-tester/mt.go

```diff
@@ -51,6 +52,7 @@ func init() {
 	flag.StringVar(&readFileContentPath, "file_content", "", "Path to read the file content from")
 	flag.StringVar(&readFileContentInLoopPath, "file_content_in_loop", "", "Path to read the file content in loop from")
 	flag.IntVar(&retryDuration, "retry_time", 180, "Retry time during the loop")
+	flag.BoolVar(&breakOnExpectedContent, "break_on_expected_content", true, "Break on expected content, (use with --file_content_in_loop flag only)")
```

Comment:
nit: maybe change the flag description to "Break out of loop on expected content ..." just to be a little clearer about what "break" means.